docs/src/main/docbook/uris-and-links.xml

```diff
@@ -63,7 +63,7 @@ public class UsersResource {
     @GET
     @Produces("application/json")
     public JSONArray getUsersAsJsonArray() {
-        JSONArray uriArray = new JSONArray();
+        JSONArray uriArray = new JSONArray  ();
```

Comment:
This is a checkstyle violation:  '(' is preceded with whitespace.

Reply:
reverted